### DISCUSSION TITLE
Terriamap is not showing

### DISCUSSION THREAD

**Discussion by GenEarly** [TerriaJS/terriajs#6772] [Q&A]
I was able to start Terriamap, but the screen is not showing.
![スクリーンショット 2023-05-09 12 28 43（2）](https://user-images.githubusercontent.com/132327338/236986711-e8cc48b5-3f98-44c8-b3c1-b58539e2ca92.png)

I would like to know how to display it.
Thank you in advance.

**Comment by na9da** [comment]
One possibility is that you might have missed the step to build the map.
Try running `yarn gulp` to build the map, then run `yarn start` to run the server.

More details are available in the [doc here](https://docker-docs--terriajs-docs-v8.netlify.app/guide/customizing/cloning-and-building/#installing-dependencies).
